### PR TITLE
Semi split use runtime

### PR DESCRIPTION
**Pull request by jpihl**
@petya2164 @mortenvp any comments?

### CODE REVIEW

File: src/kodoc/factory_binding.hpp [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+#pragma once
+
+#include <kodo/api/api.hpp>
+
+namespace kodoc
+{
+    template<class Factory>
+    using factory_binding =
+            kodo::api::storage_block_info_binding<
```
Comment (petya2164) [inline src/kodoc/factory_binding.hpp]: The indentation seems off here.

File: src/kodoc/runtime_decoder.hpp [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+#include <kodo/runtime/use_shallow_decoder_storage.hpp>
+#include <kodo/runtime/use_trace_enabled.hpp>
+
+#include "factory_binding.hpp"
+
+namespace kodoc
+{
+    template<class Stack>
+    using decoder_binding =
+            kodo::api::storage_binding<
```
Comment (petya2164) [inline src/kodoc/runtime_decoder.hpp]: This is also a double indent.

File: src/kodoc/runtime_encoder.hpp [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+#include <kodo/runtime/use_shallow_encoder_storage.hpp>
+#include <kodo/runtime/use_trace_enabled.hpp>
+
+#include "factory_binding.hpp"
+
+namespace kodoc
+{
+    template<class Stack>
+    using encoder_binding =
+            kodo::api::storage_binding<
```
Comment (petya2164) [inline src/kodoc/runtime_encoder.hpp]: Is this a wrong setting in the text editor?

**Comment by petya2164** [comment]
I think this branch is ready for merging ;)

**Comment by jpihl** [comment]
Well I'll merge it then :) :confetti_ball: Finally :confetti_ball:

**Pull request merged**